website/mails/mails.py

```diff
@@ -480,3 +480,28 @@ def get_english_article(word):
     'institution_deactivation',
     subject='Your OSF login has changed - here\'s what you need to know!'
 )
+
+REGISTRATION_BULK_UPLOAD_PRODUCT_OWNER = Mail(
+    'registration_bulk_upload_product_owner',
+    subject='Here goes the email subject'
```

Comment:
Fyi, added "place holder" email templates with contents pending. This applies to five new email and there templates.